src/main/osd/osd.c

```diff
@@ -464,11 +468,13 @@ static bool isSomeStatEnabled(void)
 
 static void osdShowStats(uint16_t endBatteryVoltage)
 {
-    uint8_t top = 2;
+    uint8_t top = 1;    /* first fully visible line */
```

Comment:
This change seems to be entirely unrelated to the odometer stats changes. This should probably go into a separate pull request.

Reply:
Moved to: https://github.com/betaflight/betaflight/pull/8022